ui/app/styles/pki/pki-beta.scss

```diff
@@ -0,0 +1,12 @@
+.pki-beta-modal-list {
+  list-style: disc;
+  padding: inherit;
+}
```

Comment:
`ul` is being overwritten globally so that default list styles do not show up. In order to show bullet points for lists, I needed to add these styles